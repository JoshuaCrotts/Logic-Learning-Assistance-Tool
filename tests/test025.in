(u)(Mu → ((Bu ∨ (Iu ∨ Hu)) & ~(Bu & (Iu ∨ Hu))))
