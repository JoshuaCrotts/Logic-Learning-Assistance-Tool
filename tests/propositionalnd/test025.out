1: (Q & P)                       P
2: R                             P
3: Q                             Derived from [(Q & P)], &E
4: P                             Derived from [(Q & P)], &E
5: (R & Q)                       Derived from [R, Q], &I
6: (P & (R & Q))                 Derived from [P, (R & Q)], &I
∴ (P & (R & Q))  ■